sky/adaptors/cloudflare.py

```diff
@@ -119,3 +120,26 @@ def create_endpoint():
     endpoint = 'https://' + accountid + '.r2.cloudflarestorage.com'
 
     return endpoint
+
+
+def r2_is_enabled() -> bool:
+    """Checks if Cloudflare R2 is enabled"""
+
+    accountid_path = os.path.expanduser(ACCOUNT_ID_PATH)
+    return os.path.exists(accountid_path)
+
+
+def get_credential_file_mounts(file_mounts: Dict[str, str]) -> Dict[str, str]:
```

Comment:
Can we remove the `file_mounts` arg for this method? We should try to keep this interface at parity with `cloud.get_credential_file_mounts`